ttnctl/util/config.go

```diff
@@ -29,30 +29,32 @@ const (
 // $HOME/.ttnctl.yml
 func GetConfigFile() string {
 	flag := viper.GetString("config")
+	xdg := path.Join(os.Getenv("XDG_CONFIG_HOME"), "ttnctl", "config.yml")
```

Comment:
If `XDG_CONFIG_HOME` is unset, this means `xdg` will be `ttnctl/config.yml`, right?
That will lead to an incorrect location in the `try_files` loop.